CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+<!---
+   Copyright 2018 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# Contributing to Eiffel Community
+Thank you for contributing to the Eiffel Community!
+
+The following is a set of guidelines for contributing to this repository.
+
+## How to Propose Changes
+Anyone is welcome to propose changes to the contents of this repository by creating a new [Issue](https://github.com/eiffel-community/eiffel/issues) ticket in GitHub. These requests may concern anything contained in the repo: changes to documentation, bug fixes, requests for additional information, additional event types, requests for additional examples, new featuers et cetera.
```

Comment:
Since this contributor guideline now refers to all eiffel-community projects I guess linking to the eiffel protocol project specifically might be a bit confusing.

Reply:
Thanks for the review @fredjn. You are right about your comment with regards to contributor guidelines being common across all the projects under Eiffel Community. I missed this while copying it from Eiffel repository.

I'll amend the PR and we can work on it together to formulate the right wording. We need this to be linked from all the projects as well since all of them have the contributor guideline in their repos.

Reply:
Will we risk end up having multiple places for issues, one global one per repo?

Reply:
I think that will be the case and project specific issues will go to that project's issue tracker/backlog as we don't have a central issue server like Jira or Bugzilla..

The community related issues/backlog items could be submitted to community repo and this repo could be owned by the Technical Committee. TC either needs to take an action and work on it or route it to the right project.

This is yet another item to discuss, clarify, and document.